nix/automation/jobs.nix

```diff
@@ -565,13 +570,12 @@ in {
     name = "update-proposal-mainnet-params";
     runtimeInputs = [nixpkgs.jq nixpkgs.coreutils];
     text = ''
-      # Inputs: $PAYMENT_KEY, $NUM_GENESIS_KEYS, $KEY_DIR, $COST_MODEL, $TESTNET_MAGIC
+      # Inputs: $PAYMENT_KEY, $NUM_GENESIS_KEYS, $KEY_DIR, $TESTNET_MAGIC
       PROPOSAL_ARGS=(
         "--max-block-body-size" "90112"
         "--number-of-pools" "500"
-        "--max-block-execution-units" '(40000000000,62000000)'
+        "--max-block-execution-units" '(20000000000,62000000)'
```

Comment:
Where do we find the authoritative values for this?

Reply:
This was matched to current mainnet chain params which were obtained with cardano-cli query protocol-parameters lookup.